start_integration.py

```diff
@@ -7,28 +7,14 @@
 from sms_notifservice import SmsNotifService
 
 with open('settings.json', "rb") as SFile:
```

Comment:
I propose to use the `settings_file` instead of `SFile` and `settings_json` instead of `settings_data` variable names

Reply:
Done